### PR TITLE
Pg add security to evsev2g

### PR DESCRIPTION
**Pull request by AssemblyJohn**

### CODE REVIEW

File: modules/EvseV2G/manifest.yaml [hunk 1]
```diff
@@ -71,6 +71,12 @@ provides:
     description: >-
       This module implements the ISO15118-2 implementation of
       an AC or DC charger
+requires:
+  security:
+    interface: evse_security
+    min_connections: 1
+    max_connections: 1
```
Comment (corneliusclaussen) [inline modules/EvseV2G/manifest.yaml:78]: min and max could be removed here, the default is always exaclty one

**Review by corneliusclaussen — CHANGES_REQUESTED** [review]
Build in the CI should be fixed before merging

**Review by SebaLukas — CHANGES_REQUESTED** [review]
Basically, I think the changes are good :+1: 
Since the requirement of EvseSecurity is not optional, please extend it for all config.yaml files that use EvseV2G. E.g. config-sil-dc, config-sil-two-evse etc.

**Review by Pietfried — APPROVED** [review]

**Review by SebaLukas — APPROVED** [review]
:+1:

**Review by corneliusclaussen — APPROVED** [review]

**Pull request merged**